datahub-web-react/src/app/entity/view/ViewsTable.tsx

```diff
@@ -56,6 +56,10 @@ export const ViewsTable = ({ views, onEditView }: ViewsTableProps) => {
                 emptyText: <Empty description="No Views found!" image={Empty.PRESENTED_IMAGE_SIMPLE} />,
             }}
             pagination={false}
+            style={{
+                height: 'calc(100% - 30px)',
```

Comment:
Where does 30px originate?

Reply:
after setting height: calc(100% - 11vh); to MangeView.tsx not need to add height just overflow can add the scroll.